Userland/Services/WebDriver/Session.cpp

```diff
@@ -160,6 +161,32 @@ Web::WebDriver::Response Session::close_window()
     return get_window_handles();
 }
 
+// 11.3 Switch to Window, https://w3c.github.io/webdriver/#dfn-switch-to-window
+Web::WebDriver::Response Session::switch_to_window(JsonValue& payload)
+{
+    // 1. Let handle be the result of getting the property "handle" from the parameters argument.
+    // 2. If handle is undefined, return error with error code invalid argument.
+    auto const& handle = TRY(WebContent::WebDriverConnection::get_property(payload, "handle"sv));
+
+    // TODO: 3. If there is an active user prompt, that prevents the focusing of another top-level browsing
```

Comment:
Instead of ´TODO´ I'd mark this and step 5 as ´FIXME´, all the other endpoints do that :^)